src/tree-node/index.js

```diff
@@ -112,7 +112,7 @@ class TreeNode extends PureComponent {
       clientId,
     } = this.props
     const liCx = getNodeCx(this.props)
-    const style = keepTreeOnSearch || !searchModeOn ? { paddingLeft: `${(_depth || 0) * 20}px` } : {}
+    const style = keepTreeOnSearch || !searchModeOn ? { paddingLeft: `${(_depth || 0) * 20 + (isLeaf(_children)?15:0)}px` } : {}
```

Comment:
Why 15? Why not 18 or 20?

Reply:
I tried 20 initially but the leaf nodes were still not in place. So I started trying all the numbers below 20 and 15 provided the perfect alignment. Let me know if you think otherwise.

Reply:
Full marks for that! I just feel like while it works in your case, if someone uses a different icon, it'll be out of alignment again. I'm wondering if there's a better fix for this.

Reply:
That's true!
One way is to parameterize it and take the value as a prop from the user.  Not the best of the solutions, I am guessing.
The user can't tackle it in the css since it's dependent on _depth. 
Let me know your thoughts and suggestions. Perhaps a hint from you and I can try to figure it out on my own then. Picked up front-end just a few months back and still have a lot to learn.

Reply:
Wish I had an answer that just worked! One way to tackle this would be to emit a "levelN" class on each node. One can then target `level1`, `level2`, ..., `levelN` styles using their own CSS. Out of the box, the library can define 10 levels in its base CSS and users can define more if needed. Bit tedious since you need to know how many levels you're going to need and create that many CSS classes. 

Another way would be to figure out the width of the "+/-" icon and adjust accordingly. This keeps the existing convenience (where user don't need to know how many levels they need or add CSS) but is more work on the component side. Not sure if there's a reliable way to figure out that width in the first place. 

> Picked up front-end just a few months back and still have a lot to learn.

That's amazing! Keep up the good work!

Reply:
These were some great suggestions which led me to think that a hidden icon can be added to each leaf node which would take care of the leaf padding itself. Committed a new change. Please have a look and let me know if it looks fine to you.

Reply:
Clever! I like it!

Reply:
Great! Hope to see the leaf padding issue fixed in the next publish! :)

Reply:
Can you resolve the code climate errors?

Reply:
Sure boss! I fixed the errors and made a new push just now! Let me know there's anything that's missing.

Reply:
Looks good!